src/connector/src/sink/kafka.rs

```diff
@@ -71,6 +72,25 @@ const fn _default_force_append_only() -> bool {
     false
 }
 
+#[derive(Debug, Clone, PartialEq, Display, Serialize, Deserialize, EnumString)]
+enum CompressionCodec {
+    #[serde(rename = "none")]
+    #[strum(serialize = "none")]
```

Comment:
We can add a `serialize_all` attribute to the whole `enum` like this one:

https://github.com/risingwavelabs/risingwave/blob/cb3d33b06045c2b6b5a74d9725ddb3616b7702dd/src/cmd_all/src/bin/risingwave.rs#L93

Reply:
thanks for the tip, done :smile: